docs/docs/examples/cat-videos.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Try-Along: Cat Videos'
+title: 'Quickstart: Cat Videos Example'
```

Comment:
The document should probably be renamed to docs/docs/quickstart to improve SEO :)